tests/plugins/test_kubejobs.py

```diff
@@ -29,11 +30,75 @@ def test_visualize_grafana():
             Returns:
                 None
     """
-    job_id = create_job(MANAGER_URL)
+    job_id = create_job(MANAGER_URL,1)
     grafana_url = wait_for_grafana_url_generation(VISUALIZER_URL, job_id)
     assert requests.get(grafana_url).ok
     stop_job(manager_url=MANAGER_URL, job_id=job_id)
 
+def test_scalling_up():
+    """ Tests if the Controlling is able to scale
+        Arguments:
+            None
+        Returns:
+            None
+    """
+    INITIAL_REPLICAS = 1
+    job_id = create_job(MANAGER_URL,2)
+    wait_for_job_complete(MANAGER_URL,job_id)
+    response = requests.get(MANAGER_URL + '/submissions/{}'.format(job_id))
+    replicas = response.json().get('final_replicas')
+    print(response.json())
+    assert int(replicas) > INITIAL_REPLICAS
```

Comment:
In this test you assert only based on the final_replicas value. In this case, if final_replicas is equal to initial_replicas, the test would fail, but it shouldn't since this doesn't mean the controller didn't scale up during the job execution, only that by the end of such execution the final_replicas were minimal. I think a better test should check if during any time of an execution the number of current replicas was higher than the initial replicas.